src/contracts/core/StrategyManager.sol

```diff
@@ -138,6 +138,10 @@ contract StrategyManager is
         uint256 expiry,
         bytes memory signature
     ) external onlyWhenNotPaused(PAUSED_DEPOSITS) nonReentrant returns (uint256 shares) {
+        require(
+            !creditTransfersDisabled[strategy],
+            "StrategyManager.depositIntoStrategyWithSignature: credit transfers disabled"
+        );
```

Comment:
I think this variable name is confusing, but i don't know what I'd call it...

Reply:
`depositsToDifferentAddressForbidden`, perhaps?
ideally something equivalent but shorter 😅

Reply:
ahh but it's not for deposits only, as it's also used when queuing withdrawals...
`transfersWithinEigenLayerForbidden` ? idk.

Reply:
`thirdPartyTransfersForbidden`, `transferOnBehalfForbidden`, `enableDelegatedTransfers`?
Personally think something like `thirdPartyTransfersForbidden` is most clear

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/a3471c4bb10919a8e8eb6c8feed36e394b182d4d